noxfile.py

```diff
@@ -148,6 +151,40 @@ def system(session):
         session.run("py.test", "--quiet", system_test_folder_path, *session.posargs)
 
 
+@nox.session(python=CONFORMANCE_TEST_PYTHON_VERSIONS)
+def conformance(session):
+    """Run the conformance test suite."""
+    conformance_test_path = os.path.join("tests", "conformance.py")
+    conformance_test_folder_path = os.path.join("tests", "conformance")
+
+    # Environment check: Only run tests if the STORAGE_EMULATOR_HOST is set.
+    if (
+        os.environ.get("STORAGE_EMULATOR_HOST", _DEFAULT_STORAGE_HOST)
+        == _DEFAULT_STORAGE_HOST
+    ):
+        session.skip("Set STORAGE_EMULATOR_HOST to run, skipping")
+    # Environment check: Only run tests if the environment variable is set.
+    if not os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", ""):
```

Comment:
Out of curiosity, why is this required?

Reply:
This is required in order to instantiate a Client. Checking to see if we have anything similar in python to the `NoCredentials` used in Java.

Reply:
You could use `Client.create_anonymous_client()` if you aren't going to be making real, over-the-wire requests.

Reply:
Or pass in an instance of `google.auth.credentials.AnonymousCredentials` to the client constructor.

Reply:
Awesome, thanks! I'll change to use one of those.